src/oci.c

```diff
@@ -523,6 +526,90 @@ cc_oci_attach (struct cc_oci_config *config,
 }
 
 /*!
+ * Get the home directory for the workload user
+ *
+ * \param config \ref cc_oci_config.
+ *
+ * \return home dir for user on sucess, "/" by default
+ */
+
+private gchar*
```

Comment:
`private` is used for functions that would normally be `static` but which we need to run tests on... could you add some tests for these new functions please? :)